libpdbg/dtb.c

```diff
@@ -608,6 +608,23 @@ static void close_dtb(struct pdbg_mfile *mfile)
 	}
 }
 
+static const uint16_t ODYSSEY_CHIP_ID = 0x60C0;
+static const uint8_t ATTR_TYPE_OCMB_CHIP = 75;
```

Comment:
Do we need these at file scope absolutely? Or, even at function scope would do?

Reply:
as it is C moved to file scope